backend/test_observer/controllers/docs/docs.py

```diff
@@ -14,14 +14,45 @@
 # You should have received a copy of the GNU Affero General Public License
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
-from fastapi import APIRouter
-from fastapi.responses import HTMLResponse
+from fastapi import APIRouter, Request
+from fastapi.responses import HTMLResponse, JSONResponse
 from fastapi.openapi.docs import get_swagger_ui_html
 
 
 router: APIRouter = APIRouter()
 
 
+@router.get("/openapi.json", include_in_schema=False)
+async def custom_openapi(request: Request):
```

Comment:
Since we are using SAML and our own custom permissions scheme, the scopes required for certain endpoints do not show up by default in the docs. This adds them so they are viewable on /docs.